modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/SnapshotOrdinalTimeline.scala

```diff
@@ -0,0 +1,37 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.data.{NonEmptyList, Validated}
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.Range.{mergeOverlaps, parseInput}
+
+import fs2.data.csv.{CellDecoder, DecoderError}
+import io.estatico.newtype.macros.newtype
+
+object SnapshotOrdinalTimeline {
```

Comment:
please name it lowercased, as `SnapshotOrdinalTimeline` should be the companion object to the case class that is inside. please take a look into other files how case classes are wrapped, for example in schema directory

Reply:
Renamed the object to `snapshotOrdinalTimeline`.